tracer/src/Datadog.Trace/Util/Http/HttpRequestUtils.cs

```diff
@@ -0,0 +1,30 @@
+// <copyright file="HttpRequestUtils.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+#if NETFRAMEWORK
+using System.Web.Routing;
+#endif
+#if !NETFRAMEWORK
```

Comment:
```suggestion
#else
```

Reply:
fixed in dc39c0955eb06c0336f4214f9c19c8de96cfa230